### DISCUSSION TITLE
Vorentwurf E-ID Gesetz: Einordnung

### DISCUSSION THREAD

**Discussion by StephanOdermattAcrea** [e-id-admin/open-source-community#17] [General]
Liebe E-ID GitHub Community

Mit Freude haben wir den neuen Vorentwurf des E-ID Gesetzes zur Kenntnis genommen. Die konzeptionellen Eckpunkte sind unseres Erachtens klug gewählt, das Potential enorm. 

Der Erfolg der neuen E-ID ist jedoch noch nicht garantiert. Damit es dieses Mal mit der breiten Nutzung durch die Schweizer Bevölkerung wirklich klappt, muss die Customer Experience noch konsequenter zu Ende gedacht werden. Welche beiden Stolpersteine wir konkret sehen und wie diese adressiert werden könnten, haben wir in folgendem Blog-Post zusammengefasst:

LinkedIn:
[https://www.linkedin.com/pulse/neues-e-id-gesetz-der-vorentwurf-birgt-enormes-doch-zwei-odermatt](https://www.linkedin.com/pulse/neues-e-id-gesetz-der-vorentwurf-birgt-enormes-doch-zwei-odermatt)

Acrea Website:
[https://www.acrea.com/blog/neues-e-id-gesetz-der-vorentwurf-birgt-enormes-potential-doch-zwei-stolpersteine-bleiben](https://www.acrea.com/blog/neues-e-id-gesetz-der-vorentwurf-birgt-enormes-potential-doch-zwei-stolpersteine-bleiben)

Wir sind gespannt auf Eure Gedanken zum Gesetzesvorentwurf sowie zu unserem Blogpost. 

Seitens Acrea leisten wir gerne unseren Beitrag zu diesem Generationenprojekt und freuen uns über jede Kontaktaufnahme und Zusammenarbeit rund um SSI.

**Comment by tobiaschristen** [comment]
Wir teilen die Einschätzung, dass der Vorentwurf zum E-ID Gesetz eine starke Vorlage ist.
Basierend auf den Aussagen unserer Partner und Kunden in der EU sind wir auch überzeugt, dass die EU im 2023 bzw. 2024 ihre Infrastruktur in Betrieb nehmen wird. Dies wird der Schweiz helfen, eine kompatible Lösung zu lancieren.

Aus unserer Sicht trifft Stephan's Artikel den Nagel auf den Kopf - ein automatisiertes Backup der Wallet ist essentiell.
Mit einem "nicht-technischen" Blog-Statement haben wir unsere Einschätzung zum Artikel von Stephan bezüglich des Secure Backups und des Secure Storage dargelegt: https://www.dswiss.com/de/news/e-id

Ich werde gerne unsere technischen Erkenntnisse aus unserer 15-jährigen Erfahrung im Verknüpfen von zBsp eBanking-ID oder Patienten-ID mit einem externen persönlichen Dokumenten- und Datenspeicher mit höchstem Datenschutz einbringen.

Ich schlage vor, dass ich hierzu einen eigenen Thread starte.

**Reply by christianheimann** [reply]
Besten Dank für die positive Rückmeldung. Sehr gerne einen eigenen Thread zu externen, persönlichen Dokumenten- und Datenspeichern mit höchstem Datenschutz resp. "automatisierte Backups von Wallet" starten.

**Comment by peterzingg** [comment]
Hallo, eine Frage zur Einordnung...
Gemäss Art. 10 Entwurf E-ID-Gesetz muss, wer in einem Prozess einer Person, die persönlich erscheint, die Möglichkeit bietet, die E-ID oder Teile davon vorzuweisen, dieser Person die Wahl lassen, sich stattdessen mit einem Ausweisdokument [...] auszuweisen, sofern die Anforderungen insbesondere an die Sicherheit des Prozesses auch auf diese Weise erfüllt werden können.
Was sind mögliche Use Cases, in denen die Sicherheit des Prozesses mit einem Ausweisdokument nicht erfüllt werden könnte?

**Reply by christianheimann** [reply]
Danke für die Frage. Es ist hier derzeit kein konkreter Use-Case benennbar. Die Absicht war, eine klare aber nicht absolute Regelung zu definieren, um auf zukünftige Szenarien reagieren zu können (z.B. verknüpfte digitale Nachweise, Datensparsamkeits-Aspekte). Der eigentliche Regelungsinhalt von Art. 10 zielt darauf ab, keinen Nutzungszwang der E-ID zu statuieren.

Der Text im Erläuternden Bericht führt zusätzlich aus:

> Mit dem vorliegenden Artikel soll sichergestellt werden, dass die Inhaberinnen und Inhaber einer E-ID nicht verpflichtet sind, bei Interaktionen in der physischen Welt ihre E-ID vorzuweisen. Trotz der Vorteile der E-ID geht es darum, die Möglichkeit, (physische) Ausweise vorzuweisen, nicht zu ersetzen oder auszuschliessen. Wenn die Identifizierung einer Person in einem physischen Prozess anhand eines Ausweises erfolgen kann, darf das Vorweisen der E-ID (oder Bestandteilen davon) nur als Option angeboten werden.
